python/pyproject.toml

```diff
@@ -51,8 +51,6 @@ dependencies = [
     "jinja2 ~= 3.1",
     "nest-asyncio ~= 1.6",
     "scipy>=1.15.1",
-    "websockets >= 13, < 16",
-    "aiortc>=1.9.0",
```

Comment:
why was this removed?

Reply:
they're not actually duplicate, they were first put into the realtime extra, but then we did need them installed as standard, and removing the realtime extra would be a breaking change, so we didn't want to do that. Did now remove the specifiers so we don't get mismatches in them

Reply:
What I mean is that they are actually duplicates in the `dependencies` table:

https://github.com/microsoft/semantic-kernel/blob/2534964cdf18ab97677e90a9dad733651b2ff6da/python/pyproject.toml#L25-L55